### ISSUE TITLE
Keep votes separately for different categories 

### ISSUE THREAD

**Issue by nkholski** [tricsi/upvote#35]
It's obvious that most games has been rated on desktop which makes it pointless to put an effort  to make games responsive on mobile and other adaptations. A desktop/mobile game should get scores within the desktop category if the voter is using a desktop  and mobile if on mobile (see #34). This will encourage better and working mobile games.

**Comment by salvan13** [comment]
agreed, in my opinion with this fix + #36, the games will be rewarded for their game strengths much more than before

**Comment by salvan13** [comment]
this probably imply that only games in the same categories can be paired during the voting phase
